cumulusci.yml

```diff
@@ -352,3 +352,40 @@ orgs:
         enterprise:
             config_file: orgs/enterprise.json
 
+plans:
+    existing_org:
+        slug: install
+        title: Install NPSP
+        tier: primary
+        steps:
+            1:
+                flow: dependencies
+                ui_options:
+                    deploy_pre:
+                        account_record_types:
+                            name: Account Record Types
```

Comment:
We should make these steps recommended.  Sometimes users have issues with those steps but want to deploy everything else.

Reply:
@jlantz Are you talking about the issues with deploying the NPSP_Default opportunity record type if the org doesn't have the right set of Stage picklist values? Or other issues too? We have a plan for how to avoid the former and I'd rather do that than make the step optional (leaving the box could also have issues if opportunity record types aren't enabled, right?)